### PR TITLE
Photo gallery modal added to PhotoMontage #386

### PR DESCRIPTION
**Pull request by vnugent**

### CODE REVIEW

File: src/pages/areas/[id].tsx [hunk 1]
```diff
@@ -181,19 +181,26 @@ export const getStaticProps: GetStaticProps<AreaPageProps, {id: string}> = async
...
     } catch (e) {
       console.log('Error when trying to add username to image data', e)
     }
 
+    /**
+     * Call Sirv API to get image metadata.  We should probably store metadata in the db.
+     */
+    const mediaListWithMetadata = await getImagesByFilenames(mediaListWithUsernames.map(entry => entry.mediaUrl))
+
+    const mediaMetaDict = indexBy<MediaType[]>(mediaListWithMetadata.mediaList, 'mediaId')
```
Comment (vnugent) [inline src/pages/areas/[id].tsx:196]: @clintonlunn we should probably store image width & height in the DB in the future.

**Comment by vnugent** [comment]
https://open-tacos-git-photo-gallery-386-openbeta-dev.vercel.app/areas/1db1e8ba-a40e-587c-88a4-64f5ea814b8e

Screen Shot 2023-01-02 at 5 31 19 PM

**Comment by clintonlunn** [comment]
@vnugent thanks for taking a look at these changes. i cleaned up the components a little bit and added getImagesByFilename() to the getStaticProps() method for climbs and crag [id].tsx pages like we discussed.

For what's left to do, the user should probably be routed to a single image viewer when clicking an image in the PhotoGalleryModal

**Comment by vnugent** [comment]
> For what's left to do, the user should probably be routed to a single image viewer when clicking an image in the PhotoGalleryModal

Can we merge this PR and continue the rest in future PRs?  It'd also be good to show tags linking to climbs/areas like we currently have in the montage component

**Comment by clintonlunn** [comment]
@vnugent sounds good, I did a last little cleanup, but it should be good to go if you agree.

**Pull request merged**